packages/react-components/src/components/MessageThread.tsx

```diff
@@ -1173,6 +1173,7 @@ export const MessageThreadWrapper = (props: MessageThreadProps): JSX.Element =>
             className={mergeClasses(classes.root, mergeStyles(styles?.chatContainer))}
             ref={chatScrollDivRef}
             style={{ ...createStyleFromV8Style(styles?.chatContainer) }}
+            {...useArrowNavigationGroup({ axis: 'vertical', memorizeCurrent: false })}
```

Comment:
It is considered a good practice to create hooks as constants at the top level in the body of a function component, otherwise it may lead to unexpected errors.
See more inforamtion in **https://react.dev/reference/rules/rules-of-hooks#only-call-hooks-at-the-top-level**